galaxy_ng/app/constants.py

```diff
@@ -16,3 +16,319 @@ class DeploymentMode(enum.Enum):
 )
 
 INBOUND_REPO_NAME_FORMAT = "inbound-{namespace_name}"
+
+PERMISSIONS = {
+    "galaxy.add_namespace": {
+        # Short name to display in the UI
+        "name": _("Add namespace"),
+
+        # Description to use when the permission is being assigned to a specifc object
+        "object_description": None,
+
+        # Description to use when the permission is being assigned globally
+        "global_description": "Create a new namespace.",
+
+        # Category to group the permission in the UI.
+        "ui_category": _("Collection Namespaces"),
+
+        # Model permissions
+        "has_model_permission": True,
```

Comment:
don't store `has_model_permission` here. This will get computed when the user requests the `me/` api